exasol/analytics/query_handler/query/select.py

```diff
@@ -101,17 +104,19 @@ class ModifyQuery(CustomQuery, AuditData):
     def __init__(
         self,
         query_string: str,
-        # Using DBObjectName instead of str, because counting rows of the modified table
```

Comment:
Add LogSpan class which generates unique IDs (probably uuid) when created and add optional parameter to supply a parent_span, #265